packages/anasazi/epetra/test/MVOPTester/cxx_main.cpp

```diff
@@ -139,16 +139,13 @@ int main(int argc, char *argv[])
         Indices[1] = MyGlobalElements[i]+1;
         NumEntries = 2;
       }
-      epetra_ierr = A->InsertGlobalValues(MyGlobalElements[i],NumEntries,&Values[0],&Indices[0]);
-      assert(epetra_ierr==0);
+      A->InsertGlobalValues(MyGlobalElements[i],NumEntries,&Values[0],&Indices[0]);
       // Put in the diagonal entry
-      epetra_ierr = A->InsertGlobalValues(MyGlobalElements[i],1,&two,&MyGlobalElements[i]);
-      assert(epetra_ierr==0);
+      A->InsertGlobalValues(MyGlobalElements[i],1,&two,&MyGlobalElements[i]);
     }
 
     // Finish building the epetra matrix A
-    epetra_ierr = A->FillComplete();
-    assert(epetra_ierr==0);
+    A->FillComplete();
```

Comment:
Hm, not sure if I like ignoring error codes, but it's standard Epetra user practice anyway ;-) .

Reply:
Yeah, these matrix constructions are safe.  I don't recommend this practice in general, just trying to eliminate the compiler warning for release builds of Trilinos.